gcloud/monitoring/client.py

```diff
@@ -195,6 +203,121 @@ def metric_descriptor(self, type_,
             display_name=display_name,
         )
 
+    @staticmethod
```

Comment:
The difference is that those factories actually need a `Client` instance to work, hence the use of `self`.

RE: "Why else would the static method decorator exist in the language?"

- Do you plan on having a subclass override this definition?
- Are you worried about polluting the namespace outside of the class?
- Does putting it on client improve the readability (i.e. would the code be harder to read if the method were instead a function with no connection to the `Client` class)?